docs/usage/configuration-options.md

```diff
@@ -560,6 +560,19 @@ Check out our [Node.js documentation](https://docs.renovatebot.com/node) for a c
 
 ## npm
 
+Extend this if you wish to configure rules specifically for `volta` definitions. Currently only `node` and `yarn` are supported. Example:
```

Comment:
Not correct. You extend the `npm` if you want to configure rules for `npm`. `volta` is now just a `depType`. I think let's remove this text completely for now and add an issue to properly document `npm` here in future (including volta, engines, etc).

Reply:
I don't disagree